src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+let leftMousePressed = false;
+let rightMousePressed = false;
+
+const firstPromise = new Promise(function (resolve, reject) {
+  document.addEventListener('mousedown', (ev) => {
+    if (ev.button === 0) {
+      resolve('First promise was resolved');
+      leftMousePressed = true;
+    }
+  });
+
+  setTimeout(() => {
+    if (!leftMousePressed) {
+      reject('First promise was rejected');
+    }
+  }, 3000);
+});
+
+firstPromise
+  .then((mes) => createDiv(mes, 'success'))
+  .catch((mes) => createDiv(mes, 'error'));
+
+const secondPromise = new Promise(function (resolve, reject) {
+  document.addEventListener('mousedown', (ev) => {
+    if (ev.button === 0 || ev.button === 2) {
+      resolve('Second promise was resolved');
+
+      if (ev.button === 0) {
+        leftMousePressed = true;
+      }
+
+      if (ev.button === 2) {
+        rightMousePressed = true;
+      }
+    }
+  });
+});
```

Comment:
The second promise resolves on any mouse button press, which includes both left and right buttons. This logic is correct based on the conditions provided, but ensure this aligns with the task requirements.